File: FormalBench-data/diverse/rule_15/TriangleArea.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class TriangleArea {

	public static int triangleArea(int r) {
		int area = 0;
		if (r < 0) {
			return -1;
		} else
			area = (r == 0) ? 0 : (int) Math.sqrt(3) * r * r;
		return area;
	}
}

File: FormalBench-data/FilteredData/fault_mutants/run_ukcw__uc/TriangleArea/mutants/4/c/TriangleArea.c
#include "java_arrays.h"
#include <math.h>
#include <stdint.h>

static int32_t java_mul(int32_t left, int32_t right) { return (int32_t)((uint32_t)left * (uint32_t)right); }

int32_t triangleArea(int32_t r) {
    int32_t area = INT32_C(0);
    if ((r < INT32_C(0))) {
        ;
    } else {
        if ((r == INT32_C(0))) {
            area = INT32_C(0);
        } else {
            area = java_mul(java_mul(((int32_t)(sqrt(INT32_C(3)))), r), r);
        }
    }
    return area;
}
